ansible/roles/java-base/tasks/main.yml

```diff
@@ -98,7 +98,10 @@
 - name: install java (macOS)
   become_user: administrator
   when: java.rc > 0 and os|startswith("macos")
-  package: name="{{ java_package_name }}" state=present
+  homebrew_cask:
+        name: "{{ java_package_name }}"
+        state: present
+  #package: name="{{ java_package_name }}" state=present
```

Comment:
delete?

Reply:
> LGTM, though I understand its WIP. There was more churn in inventory.yml than I expected, does it include some unrelated changes, or is it some cleanups while you are there?

Some cleanup I noticed the release section wasn't in order